We make GNU Backgammon use system-supplied DejaVu fonts instead of bundling Vera
fonts.  Thus stop building dynamically allocated paths for fonts.

Index: board3d/font3d.c
--- board3d/font3d.c.orig
+++ board3d/font3d.c
@@ -38,18 +38,14 @@ static double controlPoints[4][2];      /* 2D array st
 int
 CreateNumberFont(OGLFont *ppFont, const char *fontFile, int pitch, float size, float heightRatio)
 {
-    char *filename;
     FT_Library ftLib;
     if (FT_Init_FreeType(&ftLib))
         return 0;
 
-    filename = BuildFilename(fontFile);
-    if (!CreateOGLFont(ftLib, ppFont, filename, pitch, size, heightRatio)) {
-        outputerrf(_("Failed to create font from (%s)\n"), filename);
-        g_free(filename);
+    if (!CreateOGLFont(ftLib, ppFont, fontFile, pitch, size, heightRatio)) {
+        outputerrf(_("Failed to create font from (%s)\n"), fontFile);
         return 0;
     }
-    g_free(filename);
     return !FT_Done_FreeType(ftLib);
 }
 
